common/src/main/java/co/elastic/otel/common/SpanValue.java

```diff
@@ -272,9 +271,6 @@ private static Span unwrap(Object span) {
     if (span instanceof MutableSpan) {
       return unwrap(((MutableSpan) span).getOriginalSpan());
     }
-    if (CONTRIB_MUTABLE_SPAN_CLASS != null && CONTRIB_MUTABLE_SPAN_CLASS.isInstance(span)) {
```

Comment:
You can also remove now the `compileOnly` and `testImplementation` dependencies on `libs.contribSpanStacktrace` from the `common` project.

The span-stacktrace is now just a standard spanprocessor which doesn't need any special testing from the common project.